e2e-tests/.env.example

```diff
@@ -0,0 +1,11 @@
+### SENTRY TEST AUTH TOKEN ###
+SENTRY_TEST_AUTH_TOKEN="TEST_AUTH_TOKEN"
+
+### SENTRY TEST DSN ###
+SENTRY_TEST_DSN="TEST_DSN"
+
+### SENTRY TEST ORG ###
+SENTRY_TEST_ORG="TEST_ORG"
+
+### SENTRY TEST PROJECT ###
+SENTRY_TEST_PROJECT="TEST_PROJECT"
```

Comment:
@Lms24, @lforst -- Could we create a test project and set these env variables on the CI?

Reply:
Anecdotally we should avoid depending on any production systems for our tests. We recently stopped doing so in the main js repo because it was blocking us from cutting releases and merging a tad to often. Do you think it would be a hard requirement?

Reply:
This seems required for testing if the test project builds and runs in the production mode. When there's no valid `AUTH_TOKEN`, the build fails as the webpack plugin is already set on the `next.config` file. So, given there's no option to skip configuring `next.config` in the wizard at the moment, we can instead do either:

- Try reverting the local changes on `next.config` file before building the test project.
- Don't test the `build` and `start`. Only test if the project runs on `dev` mode which is working without the need for these.

Reply:
Right, I think we would want to test build and start so I am on board with pointing to a prod project. We should just make sure that it is not blocking 1) Releases of the wizard 2) PRs 3) Local test runs.

We can point the tests to the `sentry-javascript-sdks` org and the `sentry-wizard-e2e-tests` project. I think this can simply be via the github action yaml. Do we need any secrets? If so I can ask for them to be added.

Reply:
Cool, so adding the `AUTH_TOKEN` would be enough for now, I'll set the rest on the GA yaml.